fda/lfd.py

```diff
@@ -0,0 +1,91 @@
+import numpy as np
+
+from fda.basis import FDataBasis, Constant, _same_domain
+
+__author__ = "Pablo Pérez Manso"
+__email__ = "92manso@gmail.com"
+
+
+class Lfd:
+    """Defines the structure of a linear differential operator function system
+
+    .. math::
+        Lx(t) = b_0(t) x(t) + b_1(t) x'(x) + \dots + b_{n-1}(t) d^{n-1}(x(t)) + b_n(t) d^n(x(t))
+
+    Attributes:
+        nderiv (int): the order of the operator. It's the n coefficient in the equation above.
+
+        bwtlist (list):  A FDataBasis objects list of length either nderiv
+
+    """
+
+    def __init__(self, param=0, domain_range=(0, 1)):
```

Comment:
If `domain_range` is not given, but a FDataBasis is, we could deduce it. Also we should store it for the 0 order case.